src/Shared/Microsoft.DotNet.Web.Authentication/GitHub/GitHubClaimResolver.cs

```diff
@@ -165,36 +169,26 @@ void AddClaim(string type, string value)
                     }
                 }
 
+                string version = GetType().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
+                var client = new GitHubClient(new ProductHeaderValue(GetType().FullName, version), new InMemoryCredentialStore(new Credentials(accessToken)));
```

Comment:
This will break the tests, since we used the fact that we could override "OrganizationEndpoint" to test them, and the the fact that it used the options.Backchannel.  We should preserve that behavior.

We should also add tests for fetching multiple pages.